### PR TITLE
Documentation added for some endpoints.

### PR DESCRIPTION
**Pull request by rodrigo103**
## Summary

Packages updated without breaking changes. Pagination of teams fixed. Added Documentation for some endpoints.

## Known Issues

No known issues.

## Trello Card

https://trello.com/c/7NQc30od

### CODE REVIEW

**Review by gonziesc — APPROVED** [review]

**Review by gonziesc — DISMISSED** [review]
fix ci please

File: app/interactors/teams.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const createTeam = name => createTeamGithub(name).then(resp => resp.data);
+
+const addMemberToTeam = (teamId, username) =>
+  addMemberToTeamGithub({ teamId, username }).then(resp => resp.data);
+
+const addMaintainerToTeam = (teamId, username) =>
+  addMaintainerToTeamGithub({ teamId, username }).then(resp => resp.data);
+
+const deleteTeam = teamId => deleteTeamGithub({ teamId }).then(resp => resp.data);
```
Comment (mpicollo-wolox) [inline app/interactors/teams.js]: All the `resp.data` seems responsability of services instead of the interactors, because is an limitation of service response (All the consumers need data)

File: package.json [hunk 2]
```diff
@@ -60,8 +60,8 @@
     }
   },
   "dependencies": {
-    "@octokit/rest": "^16.28.7",
-    "auth0": "^2.19.0",
+    "@octokit/rest": "^16.33.1",
+    "auth0": "^2.20.0",
```
Comment (mpicollo-wolox) [inline package.json]: Check if all dependencies are in use and in place (dotenv is a devDepency), could be in another PR

**Review by picollomartin — CHANGES_REQUESTED** [review]
💪 Nice work !!!

Some little comments and we are ready to go ⏩

**Review by picollomartin — APPROVED** [review]
